ratelimit/exceptions.py

```diff
@@ -2,4 +2,9 @@
 
 
 class Ratelimited(PermissionDenied):
-    pass
+
+    def __init__(self, group, key, rate):
+        super(Ratelimited, self).__init__()
```

Comment:
I'm not sure what the signature is for Python's main `Exception` class, but if anything:
```suggestion
    def __init__(self, group, key, rate, *args, **kwargs):
        super(Ratelimited, self).__init__(*args, **kwargs)
```